packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts

```diff
@@ -208,6 +219,25 @@ export async function askResourceGroupInfo(
   }
 }
 
+async function getLocations(
+  ctx: SolutionContext,
+  rmClient: ResourceManagementClient
+): Promise<string[] | undefined> {
+  const credential = await ctx.azureAccountProvider!.getAccountCredentialAsync();
+  const subscriptionClient = new SubscriptionClient(credential as any);
```

Comment:
`credential as any` looks weird here, any reason to do the type cast here?

Reply:
Add check credential logic, so now cast logic is not needed.